lib/rjgit.rb

```diff
@@ -163,6 +163,12 @@ def self.diff(repository, options = {})
       RJGit.convert_diff_entries(diff_entries)
     end
 
+    def self.describe(repository, ref, options = {})
+      options = {:always => false, :long => false, :tags => false}.merge(options)
+      repo = RJGit.repository_type(repository)
+      git = RubyGit.new(repo).jgit
+      git.describe.set_target(ref).set_always(options[:always]).set_long(options[:long]).set_tags(options[:tags]).call
```

Comment:
This line (ending in `.call`) should be wrapped in a `begin rescue end` block and rescue the exceptions that `DescribeCommand` can throw (i.e., MissingObjectException, IncorrectObjectTypeException, RefNotFoundException, IOException).

Reply:
how should it handle those exceptions? return nil? raise a Ruby exception of some type (what type)?

Reply:
Returning `nil` would be fine.

Reply:
Done